src/cuga/backend/cuga_graph/nodes/cuga_agent_core/execution/todos.py

```
@@ -0,0 +1,166 @@
+        elif isinstance(input_data, list):
+            # If it's a list directly, convert each item to Todo
+            todos_list = [Todo(**todo) if isinstance(todo, dict) else todo for todo in input_data]
+        else:
+            # Fallback: try to create TodosInput
+            try:
+                if isinstance(input_data, dict):
+                    input_data = TodosInput(**input_data)
+                else:
+                    input_data = TodosInput(todos=input_data)
+                todos_list = input_data.todos
+            except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)